homeassistant/components/heatmiser/climate.py

```diff
@@ -41,36 +39,34 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the heatmiser thermostat."""
     from heatmiserV3 import heatmiser, connection
 
-    ipaddress = config.get(CONF_IPADDRESS)
-    port = str(config.get(CONF_PORT))
-    tstats = config.get(CONF_TSTATS)
+    HT = heatmiser.HeatmiserThermostat
```

Comment:
Can you please replace `HT` with a meaningful variable name that is conform with PEP.

Reply:
Thanks - have done so